conf/layer.conf

```diff
@@ -21,3 +21,5 @@ BBFILES_DYNAMIC += " \
 LICENSE_PATH += "${LAYERDIR}/licenses"
 include whitelist.inc
 include distro/include/raspberrypi_generic.conf
+#include distro/include/raspberrypi_generic_morty.conf
```

Comment:
is it possible to use bb.utils to choose right version of the config automatically?

Reply:
used DISTROOVERRIDE to handle it